packages/salesforcedx-vscode-soql/package.nls.json

```diff
@@ -1,5 +1,5 @@
 {
   "soqlCustom_soql": "SOQL Builder",
   "soql_builder_open_new": "SFDX: Create Query in SOQL Builder",
-  "soql_builder_toggle": "Toggle SOQL Builder"
+  "soql_builder_toggle": "Switch Between SOQL Builder and Text Editor"
```

Comment:
@ekapner - Here's the updated text for the toggle switch.  Here's what it looks like.

<img width="281" alt="Screen Shot 2021-01-13 at 11 13 35 AM" src="https://user-images.githubusercontent.com/599418/104520419-5696d100-55b8-11eb-8888-a4e23b998d27.png">

Reply:
It's long but it's descriptive. Thanks!